lib/scsocket.js

```diff
@@ -677,47 +677,51 @@ SCSocket.prototype._handleEventAckTimeout = function (eventObject, eventNode) {
   }
 };
 
-SCSocket.prototype._emit = function (event, data, callback) {
+SCSocket.prototype._emit = function (event, data) {
   var self = this;
```

Comment:
If we use fat arrow syntax `() => {}` for nested functions, then we don't need to use `self` anymore.